pkg/asset/installconfig/aws/regions.go

```diff
@@ -23,7 +23,7 @@ func knownRegions(architecture types.Architecture) map[string]string {
 }
 
 // IsKnownRegion return true is a specified region is Known to the installer.
-// A known region is subset of AWS regions and the regions where RHEL CoreOS images are published.
+// A known region is subset of public AWS regions and the regions where RHEL CoreOS images are published.
 func IsKnownRegion(region string, architecture types.Architecture) bool {
```

Comment:
The names of these functions should be updated to make it clear that it is specifically known public regions.